hack/update-codegen.sh

```diff
@@ -7,229 +7,45 @@ set -o pipefail
-    --openapi-schema ./vendor/github.com/openshift/api/openapi/openapi.json \
-    --trim-path-prefix github.com/openshift/client-go \
-    ${verify}
-  bash ${CODEGEN_PKG}/generate-groups.sh "client" \
-    github.com/openshift/client-go/${group} \
-    github.com/openshift/api \
-    "${group}:v1beta1" \
-    --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.txt \
-    --plural-exceptions=DNS:DNSes,DNSList:DNSList,Endpoints:Endpoints,Features:Features,FeaturesList:FeaturesList,SecurityContextConstraints:SecurityContextConstraints \
-    --apply-configuration-package github.com/openshift/client-go/${group}/applyconfigurations \
-    --trim-path-prefix github.com/openshift/client-go \
-    ${verify}
+source "${CODEGEN_PKG}/kube_codegen.sh"
+
+for group in apiserver apps authorization build cloudnetwork config console helm image imageregistry insights machine monitoring network oauth operator operatorcontrolplane project quota route samples security securityinternal servicecertsigner sharedresource template user; do
+  echo "# Processing ${group} ..."
+  kube::codegen::gen_client \
+      --with-watch \
+      --with-applyconfig \
+      --applyconfig-name "applyconfigurations" \
+      --applyconfig-externals "github.com/openshift/api/operator/v1.OperatorSpec:github.com/openshift/client-go/operator/applyconfigurations/operator/v1,github.com/openshift/api/operator/v1.OperatorStatus:github.com/openshift/client-go/operator/applyconfigurations/operator/v1,github.com/openshift/api/operator/v1.OperatorCondition:github.com/openshift/client-go/operator/applyconfigurations/operator/v1,github.com/openshift/api/operator/v1.GenerationStatus:github.com/openshift/client-go/operator/applyconfigurations/operator/v1" \
+      --input-pkg-root "github.com/openshift/api" \
+      --one-input-api "${group}" \
+      --output-pkg-root "github.com/openshift/client-go/${group}" \
+      --output-base "$(dirname "${BASH_SOURCE[0]}")/../../../.." \
+      --plural-exceptions "DNS:DNSes,DNSList:DNSList,SecurityContextConstraints:SecurityContextConstraints" \
+      --boilerplate "${SCRIPT_ROOT}/hack/boilerplate.txt"
 done
 
-for group in servicecertsigner operatorcontrolplane sharedresource insights; do
-  bash ${CODEGEN_PKG}/generate-groups.sh "lister,informer" \
-    github.com/openshift/client-go/${group} \
-    github.com/openshift/api \
-    "${group}:v1alpha1" \
-    --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.txt \
-    --plural-exceptions=DNS:DNSes,DNSList:DNSList,Endpoints:Endpoints,Features:Features,FeaturesList:FeaturesList,SecurityContextConstraints:SecurityContextConstraints \
-    --trim-path-prefix github.com/openshift/client-go \
-    ${verify}
-  generateApplyConfiguration \
-    github.com/openshift/client-go/${group} \
-    github.com/openshift/api \
-    "${group}:v1alpha1" \
-    --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.txt \
-    --openapi-schema ./vendor/github.com/openshift/api/openapi/openapi.json \
```

Comment:
This arg isn't being carried over and is causing a diff, has it been removed as an option?

Reply:
Possibly this is related with that `Extract...` function above, looking into it.